widgets/src/main/java/ru/startandroid/widgets/WidgetsConfigActivity.kt

```diff
@@ -0,0 +1,17 @@
+package ru.startandroid.widgets
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+
+class WidgetsConfigActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_widgets_config)
+        val data = intent.data
```

Comment:
Try to open this screen by calling deeplink and then rotate a device. Toast will be created again. We don't need this behaviour. It happens because intent still contains deeplink when activity is rotated or recreated. Let's remove it. 
After intent handling remove data from intent. After device rotate you will get clean intent. 
And you need to add null checking for intent.data.